runtime/help/options.md

```diff
@@ -462,23 +462,12 @@ Here are the available options:
 
 ---
 
-Plugin options: all plugins come with a special option to enable or disable
-them. The option is a boolean with the same name as the plugin itself.
-
-By default, the following plugins are provided, each with an option to enable
-or disable them:
-
-* `autoclose`: automatically closes brackets, quotes, etc...
-* `comment`: provides automatic commenting for a number of languages
-* `ftoptions`: alters some default options depending on the filetype
-* `linter`: provides extensible linting for many languages
-* `literate`: provides advanced syntax highlighting for the Literate
-   programming tool.
-* `status`: provides some extensions to the status line (integration with
-   Git and more).
-* `diff`: integrates the `diffgutter` option with Git. If you are in a Git
-   directory, the diff gutter will show changes with respect to the most
-   recent Git commit rather than the diff since opening the file.
```

Comment:
It really is a question, but I believe it would be better to redirect the user to the plugin documentation to keep the same text in different places. What do you think @dmaluka?

Reply:
I think: it is clearly up to discussion, so if you really want to do it, I'd suggest doing it in a separate PR.

Reply:
@JoeKar I'm going to return options.md to the old format, perhaps it would be better to create an issue to formalize the problem with the community and a way to fix it.